code.json

```diff
@@ -84,6 +84,50 @@
       "email": "jlaura@usgs.gov"
     },
 
+    "date": {
+      "metadataLastUpdated": "2023-04-19"
+    }
+  },{
+    "name": "ISIS3",
+    "organization": "U.S. Geological Survey",
+    "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
+    "version": "8.1.0_RC1",
+    "status": "Production",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/isis/-/raw/7.1.0/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://isis.astrogeology.usgs.gov",
+    "downloadURL": "https://code.usgs.gov/astrogeology/isis/-/archive/8.1.0_RC1/8.1.0_RC1.zip",
+    "disclaimerURL": "https://code.usgs.gov/astrogeology/isis/-/raw/8.1.0_RC1/DISCLAIMER.md",
+    "repositoryURL": "https://code.usgs.gov/astrogeology/isis.git",
+    "vcs": "git",
+
+    "laborHours": 0,
+
+    "tags": [
+      "Planetary",
+      "Remote Sensing",
+      "Photogrammetry",
+      "Data Processing"
+    ],
+
+    "languages": [
+      "C++"
+    ],
+
+    "contact": {
+      "name": "Dr. Jason Laura",
+      "email": "jlaura@usgs.gov"
```

Comment:
This should probably be someone on the dev team. I think per release we just make it the person who made the release